### PR TITLE
custom email templates selection in selection plan crud

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=2875188&fileview=grid

### CODE REVIEW

File: src/pages/summits/edit-selection-plan-page.js [hunk 1]
```diff
@@ -35,6 +36,10 @@ class EditSelectionPlanPage extends React.Component {
         this.onUpdateExtraQuestionOrder = this.onUpdateExtraQuestionOrder.bind(this);
     }
 
+    componentDidMount() {
+        this.props.getEmailTemplates();
```
Comment (smarcet) [inline src/pages/summits/edit-selection-plan-page.js:40]: please dont get all also this is paginating and you are getting the first ones

File: src/components/forms/selection-plan-form.js [hunk 2]
```diff
@@ -309,61 +335,94 @@ class SelectionPlanForm extends React.Component {
...
+                                    <Dropdown
+                                        id="creator_notification_email_id"
+                                        value={entity.creator_notification_email_template}
+                                        onChange={this.handleNotificationEmailTemplateChange}
+                                        options={emailNotificationTemplatesDDL}
+                                    />
+                                </div>
+                                <div className="col-md-6">
+                                    <label> {T.translate("edit_selection_plan.moderator_notification_email_template")}</label>
+                                    <Dropdown
```
Comment (smarcet) [inline src/components/forms/selection-plan-form.js:405]: please use same control as https://github.com/fntechgit/summit-admin/blob/0af7876e819fa61934cb8acd57d0db38d454c753/src/components/forms/email-flow-event-form.js#L89

File: src/components/forms/selection-plan-form.js [hunk 3]
```diff
@@ -309,61 +335,94 @@ class SelectionPlanForm extends React.Component {
...
+                        </Panel>
+                        <Panel
+                            show={showSection === 'email_templates'}
+                            title={T.translate("edit_selection_plan.email_templates")}
+                            handleClick={() => {this.toggleSection('email_templates')}}
+                        >
+                             <div className="row form-group">
+                                <div className="col-md-6">
+                                    <label> {T.translate("edit_selection_plan.creator_notification_email_template")}</label>
+                                    <Dropdown
```
Comment (smarcet) [inline src/components/forms/selection-plan-form.js:396]: please use same control as https://github.com/fntechgit/summit-admin/blob/0af7876e819fa61934cb8acd57d0db38d454c753/src/components/forms/email-flow-event-form.js#L89

File: src/components/forms/selection-plan-form.js [hunk 4]
```diff
@@ -309,61 +335,94 @@ class SelectionPlanForm extends React.Component {
...
+                                        value={entity.moderator_notification_email_template}
+                                        onChange={this.handleNotificationEmailTemplateChange}
+                                        options={emailNotificationTemplatesDDL}
+                                    />
+                                </div>
+                            </div>
+                            <div className="row form-group">
+                                <div className="col-md-6">
+                                    <label> {T.translate("edit_selection_plan.speaker_notification_email_template")}</label>
+                                    <Dropdown
```
Comment (smarcet) [inline src/components/forms/selection-plan-form.js:416]: please use same control as https://github.com/fntechgit/summit-admin/blob/0af7876e819fa61934cb8acd57d0db38d454c753/src/components/forms/email-flow-event-form.js#L89

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: src/components/forms/selection-plan-form.js (hunk 3 above)
Comment (romanetar) [reply src/components/forms/selection-plan-form.js]: done

File: src/components/forms/selection-plan-form.js (hunk 2 above)
Comment (romanetar) [reply src/components/forms/selection-plan-form.js]: done

File: src/components/forms/selection-plan-form.js (hunk 4 above)
Comment (romanetar) [reply src/components/forms/selection-plan-form.js]: done

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**